code/game/objects/items/policetape.dm

```diff
@@ -260,6 +261,7 @@
 // -- /taperoll/syndie = contains all the things dealing with charges
 /obj/item/taperoll/syndie
 	var/charges_left = 12
+	mech_flags = MECH_SCAN_ILLEGAL
```

Comment:
wait, that's illegal.

Reply:
The syndie tape, yes. Normal tape is not illegal.